cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -34,11 +34,26 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	apiserveroptions "k8s.io/apiserver/pkg/server/options"
 	componentbaseconfig "k8s.io/component-base/config"
+	configv1alpha1 "k8s.io/component-base/config/v1alpha1"
 	"k8s.io/component-base/logs"
+	configv1beta1 "k8s.io/kube-scheduler/config/v1beta1"
```

Comment:
nit: can we have these without aliases?
Or at least the aliases don't have to say "kubescheduler", given the location of the file, if there are no conflicts.